### ISSUE TITLE
More featured projects on the main page

### ISSUE THREAD

**Issue by okdistribute** [whoisgina/dat-foundation-website#157]
Right now it only shows 2 featured projects on the main page. Let's add more to that list. Hopefully this is only a very tiny change in the JavaScript or HTML that places a limit on the number that are shown.

**Comment by RangerMauve** [comment]
Yeah! We just need to modify the `featured` flag to True in any of the projects YAML files. https://github.com/datproject/dat-foundation-website/blob/release/site/projects/beaker.md
